src/pymor/models/interface.py

```diff
@@ -104,7 +104,7 @@ def _compute_output(self, solution, mu=None, **kwargs):
         must contain the key `'output'`.
         """
         if not getattr(self, 'output_functional', None):
-            return np.zeros(len(solution), 0)
+            return np.zeros(len(solution))
```

Comment:
That should be

```suggestion
            return np.zeros((len(solution),0))
```

as we have a zero-dimensional output space.

Reply:
Ah right, thanks. I should have spent another minute to think about this....